.circleci/config.yml

```
@@ -0,0 +1,47 @@
+# Scala CircleCI 2.0 configuration file
+# See: https://circleci.com/docs/2.0/sample-config/
+version: 2
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/2.0/configuration-reference/#jobs
+jobs:
+  build:
+    # Specify the execution environment. You can specify an image from Dockerhub or use one of our Convenience Images from CircleCI's Developer Hub.
+    # See: https://circleci.com/docs/2.0/configuration-reference/#docker-machine-macos-windows-executor
+    docker:
+      # specify the version you desire here
+      - image: circleci/openjdk:8-jdk
```

Circle example is already outdated, renovate the image

```suggestion
      - image: cimg/openjdk:19.0.2
```